.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.12.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.12.x'
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Go Get
+        run: |
+          go get
```

Comment:
what does this first `go get` do?

Reply:
Line 24 is the name of this run step but the first `go get` run in the command downloads and compiles the dependencies in `go.mod`. I kept this the same as in Travis even though I think `go install` is now [recommended](https://tip.golang.org/doc/go1.16#modules). But I am not entirely sure.